pipeline/restart_node.sh

```diff
@@ -319,7 +319,7 @@ fetch_binaries() {
 	tf)
 		rn_info "fetching upgrade binaries on tf node"
 		node_ssh "${ip}" "
-			./node.sh -U upgrade -d
+			./node.sh -U upgrade -I -d
```

Comment:
Looks good. do we need to add an explanation for `-I`  in node.sh usage as well?

Reply:
https://github.com/harmony-one/harmony/blob/5878439679077b2165b8dbd6153bc4306859ce5a/scripts/node.sh#L120